types/pragma/index.d.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Parse pragma and its content from JavaScript comments
+ *
+ * @param {string} input
+ * @param {{parseContent?: (string) => any}} option
+ * @return {object}
+ */
+declare function pragma(input: string, option?: { parseContent?: (source: string) => any }): object;
```

Comment:
FYI, the `object` type doesn't refer exclusively to objects like `{}`; it also includes arrays and class instances.

You might want something like this:

```suggestion
declare function pragma(input: string, option?: { parseContent?: (source: string) => any }): Record<string, any>;
```